cmd/register.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright © 2022 Netmaker Team <info@netmaker.io>
+*/
+package cmd
+
+import (
+	"github.com/gravitl/netclient/functions"
+	"github.com/gravitl/netmaker/logger"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// registerCmd represents the register command
+var registerCmd = &cobra.Command{
+	Use:   "register",
+	Short: "register to a Netmaker instance",
+
+	Long: `register to a Netmaker instance using: 
+token: netclient register -t <token> // join using an enrollment token`,
+	Run: func(cmd *cobra.Command, args []string) {
+		flags := viper.New()
```

Comment:
lines 21- 29 should be
token, err := cmd.Flags().GetString("token")
if err != nil {
    cmd.Usage()
    return
}